decentraland/unity-renderer
# feat: Relocate description text in chat on resize

## What does this PR change?

When messages fill up the chat view, relocates the description of the corresponding chat view into the scroll. For `PrivateChat` and `ChannelChat`, there is another analysis on each chat container reset. If the new messages content dont fill up, the description is relocated at its original position

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat%2Frelocate-description-text-on-masked&realm=dg&position=100%2C100
2. Open the general chat.
3. Write messages until it fills the chat window. Check that the description starts going up when filled.
4. Repeat 2 and 3 for private chat window and channel chat window
5. Close and open new a channel chat window. Check that the message goes back to the original position if the new channel content doesnt fill up the window.
6. Repeat 5 for private chat window

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
